.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+source = src
+
+[report]
+exclude_lines =
+    pragma: no cover
+    raise NotImplementedError
+    if __name__ == '__main__':
```

Comment:
I guess we could do that. I'm personally pretty used to .coveragerc though and would prefer not to; one minor reason is that one or the other of the files may be customized in a series of projects while the other is identical, so it's easier to copy around if the files are separate (I admit that's not an especially great reason).